{"hierarchy":{"paths":[["doc:\/\/co.pointfree.VaporRouting\/documentation\/NIOWebSocket","doc:\/\/co.pointfree.VaporRouting\/documentation\/NIOWebSocket\/WebSocketErrorCode"]]},"variants":[{"traits":[{"interfaceLanguage":"swift"}],"paths":["\/documentation\/niowebsocket\/websocketerrorcode\/unexpectedservererror"]}],"schemaVersion":{"patch":0,"major":0,"minor":3},"metadata":{"title":"WebSocketErrorCode.unexpectedServerError","role":"symbol","fragments":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"kind":"identifier","text":"unexpectedServerError"}],"roleHeading":"Case","modules":[{"name":"NIOWebSocket"}],"externalID":"s:12NIOWebSocket03WebB9ErrorCodeO016unexpectedServerD0yA2CmF","symbolKind":"case"},"primaryContentSections":[{"kind":"declarations","declarations":[{"languages":["swift"],"platforms":["Linux"],"tokens":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"kind":"identifier","text":"unexpectedServerError"}]}]}],"identifier":{"url":"doc:\/\/co.pointfree.VaporRouting\/documentation\/NIOWebSocket\/WebSocketErrorCode\/unexpectedServerError","interfaceLanguage":"swift"},"kind":"symbol","sections":[],"abstract":[{"text":"Indicates that a server is terminating the connection because","type":"text"},{"text":" ","type":"text"},{"text":"it encountered an unexpected condition that prevented it from","type":"text"},{"text":" ","type":"text"},{"text":"fulfilling the request.","type":"text"},{"text":" ","type":"text"},{"text":"Corresponds to code 1011.","type":"text"}],"references":{"doc://co.pointfree.VaporRouting/documentation/NIOWebSocket/WebSocketErrorCode":{"kind":"symbol","fragments":[{"kind":"keyword","text":"enum"},{"kind":"text","text":" "},{"kind":"identifier","text":"WebSocketErrorCode"}],"url":"\/documentation\/niowebsocket\/websocketerrorcode","navigatorTitle":[{"text":"WebSocketErrorCode","kind":"identifier"}],"identifier":"doc:\/\/co.pointfree.VaporRouting\/documentation\/NIOWebSocket\/WebSocketErrorCode","title":"WebSocketErrorCode","abstract":[{"type":"text","text":"An enum that represents websocket error codes."}],"type":"topic","role":"symbol"},"doc://co.pointfree.VaporRouting/documentation/NIOWebSocket":{"kind":"symbol","type":"topic","abstract":[],"role":"collection","url":"\/documentation\/niowebsocket","identifier":"doc:\/\/co.pointfree.VaporRouting\/documentation\/NIOWebSocket","title":"NIOWebSocket"},"doc://co.pointfree.VaporRouting/documentation/NIOWebSocket/WebSocketErrorCode/unexpectedServerError":{"type":"topic","identifier":"doc:\/\/co.pointfree.VaporRouting\/documentation\/NIOWebSocket\/WebSocketErrorCode\/unexpectedServerError","title":"WebSocketErrorCode.unexpectedServerError","kind":"symbol","fragments":[{"text":"case","kind":"keyword"},{"text":" ","kind":"text"},{"text":"unexpectedServerError","kind":"identifier"}],"role":"symbol","abstract":[{"text":"Indicates that a server is terminating the connection because","type":"text"},{"text":" ","type":"text"},{"text":"it encountered an unexpected condition that prevented it from","type":"text"},{"text":" ","type":"text"},{"text":"fulfilling the request.","type":"text"},{"text":" ","type":"text"},{"text":"Corresponds to code 1011.","type":"text"}],"url":"\/documentation\/niowebsocket\/websocketerrorcode\/unexpectedservererror"}}}